packages/components/src/internal/renderers/FileColumnRenderer.tsx

```diff
@@ -20,15 +20,76 @@ import { FILELINK_RANGE_URI } from '../components/domainproperties/constants';
 
 import { QueryColumn } from '../../public/QueryColumn';
 
-import { AttachmentCard, IAttachment } from './AttachmentCard';
+import { AttachmentCard, AttachmentCardProps, IAttachment } from './AttachmentCard';
 
-interface Props {
+interface OwnProps {
     col?: QueryColumn;
     data?: any;
     onRemove?: (attachment: IAttachment) => void;
 }
 
-export class FileColumnRenderer extends PureComponent<Props> {
+interface FileProp {
+    fileUnavailable?: boolean;
+    filename: string;
+}
+
+const getFileDisplayValue = (rawDisplayValue: string): FileProp => {
+    if (!rawDisplayValue) {
+        return {
+            filename: undefined,
+            fileUnavailable: false, // not a file_link, could be attachment
+        };
+    }
+
+    if (rawDisplayValue.endsWith(' (unavailable)')) {
```

Comment:
minor: stash as something like `const UNAVAILABLE_SUFFIX = ' (unavailable);` and then reuse a few lines down

Reply:
Also, it seems like if you just changed this to add a null check for rawDisplayValue, then you could remove the if statement above and let it fall through to the return at line 50:
```
if (rawDisplayValue?.endsWith(' (unavailable)')) {
```